spec/defines/haproxy_service_spec.rb

```diff
@@ -49,13 +49,13 @@
 
         [
           "frontend svc1-hatcher-http-front\n" \
-          "  bind 1.2.3.4:80,40.41.42.43:80\n" \
+          "  bind 1.2.3.4:80\n" \
           "  stats uri \/haproxy?stats\n" \
           "  default_backend svc1-hatcher-http-back\n" \
           "  http-request set-header X-Client-IP %ci\n" \
           "  http-request set-header X-Forwarded-Proto http\n",
           "frontend svc1-hatcher-https-front\n" \
-          "  bind 1.2.3.4:443,40.41.42.43:443 ssl crt /etc/ssl/private/svc1\n" \
+          "  bind 1.2.3.4:443 ssl crt /etc/ssl/private/svc1\n" \
```

Comment:
I'm not sure if I was right to change the tests like this—but it made them pass. It looked to me like you intentionally removed the local ips from the template, so I removed them from the tests as well.

Reply:
Yes, that was exactly the right thing to do - thanks